freqtrade/rpc/telegram.py

```diff
@@ -115,7 +115,7 @@ def _init_keyboard(self) -> None:
                                  r'/stopbuy$', r'/reload_config$', r'/show_config$',
                                  r'/logs$', r'/whitelist$', r'/blacklist$', r'/bl_delete$',
                                  r'/weekly$', r'/weekly \d+$', r'/monthly$', r'/monthly \d+$',
-                                 r'/forcebuy$', r'/forcelong$', r'/forceshort$',
+                                 r'/forcebuy$', r'/forcelong$', r'/forceshort$', r'/forceexit$',
```

Comment:
This is wrong here at this point in time.

issuing `/forceexit` without any parameter will not do anything other than showing a warning that either "all" or a trade number must be specified.